### PR TITLE
Update for another suite patterns branch

### PR DESCRIPTION
**Pull request by wiiitek**
Some code changes after feature testing

### CODE REVIEW

File: core/datastorage/src/main/java/com/cognifide/aet/vs/metadata/DocumentConverter.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
 class DocumentConverter {
 
   private static final Gson GSON_FOR_MONGO_JSON = new GsonBuilder()
+      //.registerTypeHierarchyAdapter(Optional.class, new OptionalDeserializer())
```
Comment (subiron) [inline core/datastorage/src/main/java/com/cognifide/aet/vs/metadata/DocumentConverter.java]: why do we want to leave this line here?

Comment (wiiitek) [inline core/datastorage/src/main/java/com/cognifide/aet/vs/metadata/DocumentConverter.java]: To be removed, thanks!

File: pom.xml [hunk 2]
```diff
@@ -601,6 +601,9 @@
           <groupId>com.cognifide.aet</groupId>
           <artifactId>aet-maven-plugin</artifactId>
           <version>${project.version}</version>
+          <configuration>
```
Comment (subiron) [inline pom.xml]: is it needed?

Comment (wiiitek) [inline pom.xml]: yes, it is used when we want to specify different pattern suite from command line.
then we overwrite this empty value with:
```
-DpatternSuite=demo
```

**Pull request merged**